tests/conftest.py

```diff
@@ -16,6 +49,75 @@ def pytest_addoption(parser):
     )
 
 
+@pytest.fixture(scope="function")
+def shallow_org_tree():
+    """Create shallow org tree for common test needs"""
+    org_101 = Organization(id=101, name='101')
+    org_102 = Organization(id=102, name='102')
+    org_1001 = Organization(id=1001, name='1001', partOf_id=101)
+
+    already_done = Organization.query.get(101)
+    if already_done:
+        return
+
+    with SessionScope(db):
+        [db.session.add(org) for org in (org_101, org_102, org_1001)]
+        db.session.commit()
+
+    # As orgs were just added, make sure they're loaded on next OrgTree call
+    OrgTree.invalidate_cache()
+
+    yield  # setup complete, teardown follows
+
+    # After using this fixture, invalidate OrgTree so subsequent tests
+    # don't find stale entries, namely dead references to the orgs created
+    # above.
+    OrgTree.invalidate_cache()
+
+
+def calc_date_params(backdate, setdate):
+    """
+    Returns the calculated date given user's request
+
+    Tests frequently need to mock times, this returns the calculated time,
+    either by adjusting from utcnow (via backdate), using the given value
+    (in setdate) or using the value of now if neither of those are available.
+
+    :param backdate: a relative timedelta to move from now
+    :param setdate: a specific datetime value
+    :return: best date given parameters
+
+    """
+    if setdate and backdate:
+        raise ValueError("set ONE, `backdate` OR `setdate`")
+
+    if setdate:
+        acceptance_date = setdate
+    elif backdate:
+        # Guard against associative problems with backdating by
+        # months.
+        if backdate.months:
+            raise ValueError(
+                "moving dates by month values is non-associative; use"
+                "`associative_backdate` and pass in `setdate` param")
+        acceptance_date = datetime.utcnow() - backdate
+    else:
+        acceptance_date = datetime.utcnow()
+    return acceptance_date
+
+
+@pytest.fixture
+def initialize_static(initialized_db):
+    add_static_concepts(only_quick=True)
+    add_static_interventions()
+    add_static_organization()
+    add_static_relationships()
+    add_static_roles()
+    db.session.commit()
+
+    yield
```

Comment:
The point of a `yield` in a fixture, is to mark a point of return, so after the test runs, context pops back to the fixture for clean up.  It's likely to be confusing to leave a yield at the end of a fixture without cleanup code following.  See https://pytest.readthedocs.io/en/2.9.1/yieldfixture.html for more info.